src/yadism/coefficient_functions/light/nnlo/g1.py

```diff
@@ -53,41 +65,82 @@ def gluon_reg(z, args):
     lnz3 = lnz**3
     ln1pz = np.log(1 + z)
     ln1pz2 = ln1pz**2
-    # fmt: off
-    return nf * (
-        CF * TR * (
-            ( 1 - 2 * z ) * ( 
-                32 * Li31mz - 16 * ln1mz * Li21mz - 8 * lnz * Li21mz - 24 * zeta2 * lnz
-                - 20 * ln1mz3 / 3.0+ 16 * lnz * ln1mz2 - 16 * lnz2 * ln1mz + 10 * lnz3 / 3
+
+    gluon_regular = nf * (
```

Comment:
This variable is not used, other than returning it, and it is the exact name of the function.

Since this is not Fortran code, feel free to immediately return:
```suggestion
    return nf * (
```

Reply:
Since the expressions are so long, having a variable would be more helpful in terms of debugging, but fine given that it is just a copy & paste I guess that won't hurt.